contract/build.sh

```diff
@@ -0,0 +1 @@
+env 'RUSTFLAGS=-C link-arg=-s' cargo build --target wasm32-unknown-unknown --release
```

Comment:
this file should be removed and we should use Makefile to make sure we all use same scripts.

Reply:
ok deleted